src/cljc/quil/snippets/math/trigonometry.cljc

```diff
@@ -14,6 +14,22 @@
   (q/text (str "(q/acos 0) = " (q/acos 0)) 10 20)
   (q/text (str "(q/acos 1) = " (q/acos 1)) 10 40))
 
+#?(:cljs
+    (defsnippet angle-mode
+      "angle-mode"
+      {}
+
+      (q/angle-mode :degrees)
+
+      (q/fill "red")
+      (q/rect 50 50 100 100)
+
+      (comment "rotation will be in degrees")
+      (q/rotate q/TWO-PI)
```

Comment:
If it's in degrees, maybe use something like 45? TWO-PI sounds like radians and might be confusing. Also do you mind adding example with (q/angle-mode :radians) for completeness?

Reply:
👍